scripts/jenkins/docker-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -ex
+export GOPATH=$WORKSPACE
+eval "$(gvm $GO_VERSION)"
+go get -v -u github.com/jstemmer/go-junit-report
+go get -v -u github.com/axw/gocov/gocov
+go get -v -u github.com/matm/gocov-html
+go get -v -u github.com/axw/gocov/...
+go get -v -u github.com/AlekSi/gocov-xml
+
+go get -v -t ./...
+
+export COV_FILE="build/coverage.cov"
+export OUT_FILE="build/test-report.out"
+mkdir -p build
+
+./scripts/docker-compose-testing up -d --build
+./scripts/docker-compose-testing run -T --rm go-agent-tests make coverage | tee ${COV_FILE}.raw
+echo "mode: atomic" > ${COV_FILE}
```

Comment:
I think gocov (which I wrote before `go test -cover` existed) can be replaced to make this a bit simpler, cutting down the number of external dependencies.

- https://github.com/t-yuki/gocover-cobertura can convert ${COV_FILE}.raw into XML output for the Cobertura plugin.
- `go tool cover -html=${COV_FILE}.raw -o build/coverage-apm-agent-go-docker-report.xml` will produce an HTML report

Reply:
IMHO the html coverage output of [gocov-xml](https://github.com/matm/gocov-html) than the default one, we can open an issue related with that and let the whole team decided.

what is the difference/benefit to use [gocover-cobertura](https://github.com/t-yuki/gocover-cobertura) instead [gocov-xml](https://github.com/AlekSi/gocov-xml)? I saw that gocover-cobertura is updated more recently, I do not have a strong opinion the tool that you said is better, it is better for me ;)

Reply:
> IMHO the html coverage output of gocov-xml than the default one, we can open an issue related with that and let the whole team decided.

Sure, we can put it up for a vote. I'd just prefer to use more of the standard tooling, makes it easier for newcomers. Having fewer tools means less to maintain, too.

> what is the difference/benefit to use gocover-cobertura instead gocov-xml? I saw that gocover-cobertura is updated more recently, I do not have a strong opinion the tool that you said is better, it is better for me ;)

The main benefit is that gocover-cobertura doesn't require gocov, it uses the output of the built-in "go tool -cover".

Reply:
fair enough, you convinced me ;)